GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -85,14 +85,11 @@ public override IEnumerable<EventListener> CreateTelemetryListeners(string provi
                 yield return etwListener;

```

Comment:
Will we eventually remove this listener?  Is there anything that will be preventing double entries in reporting?

Reply:
There's actually a chance of triple reporting here.
We will eventually remove the ETW listener on Windows, since the telemetry daemon on Windows can post to both ETW and AI any messages it gets.

@jrbriggs what do we want to do here? There are 3 telemetry paths on Windows:
1. ETW in-proc
2. ETW via pipe/service
3. AI via pipe/service

(on Mac we only have path 3)

Do we want to run the in-proc ETW listener on Windows until such a time that we're happy either the telemetry pipe is reliable, and/or the ETW publishing in the daemon is good, and/or the unified AI->kusto process works?

Reply:
I want to have VFS for Git work consistently on each platform. I think this means that we cut over to the daemon publishing. That will avoid duplicates going through the ETW pipeline.

Let's not publish the VFS4G stream to AI in this release.

Reply:
> Let's not publish the VFS4G stream to AI in this release.

But then we'd have nothing coming from Mac? Do you mean:

On Windows:

- disable ETW in-proc listener
- enable telemetry pipe listener
- configure telemetry service to post to ETW only

On Mac:

- enable telemetry pipe listener
- configure telemetry service to post to AI only

Reply:
The 1.0.0.4 release of GitService that we're shipping in this cycle
does not have any of the AI nor VFS proxy code.  This version only
does ETW and only for Trace2 traffic.

We need to wait until next sprint to cut over.  (Or pull more code
over from my master-candidate branch into this maint branch.)

Reply:
I've removed the ETW in-proc listener completely.